lib/create_api_gem/forms/blocks/dropdown_block.rb

```diff
@@ -18,16 +18,17 @@
 require_relative 'block'
 
 class DropdownBlock < Block
-  attr_accessor :id, :title, :type, :ref, :description, :alphabetical_order, :choices, :required, :attachment
+  attr_accessor :id, :title, :type, :ref, :description, :alphabetical_order, :randomize, :choices, :required, :attachment
 
   def initialize(id: nil, title: nil, type: :dropdown, ref: nil, description: nil, alphabetical_order: nil,
-                 choices: nil, required: nil, attachment: nil)
+                 randomize: nil, choices: nil, required: nil, attachment: nil)
```

Comment:
Does the order of arguments matter here (in ruby)? Won't it break for existing users if we put `randomize` in the middle (i.e. not making it the last argument)? Maybe it makes sense to write a test for that.

Reply:
Yeah, now that you say that, I think you are right. I'll move it to the end.